src/BlockchainTestsFiller/InvalidBlocks/bcStateTests/ZeroValue_TransactionCALL_ToEmpty_OOGRevertFiller.json

```diff
@@ -23,7 +23,7 @@
 	    "sealEngine" : "NoProof",
         "expect" : [
             {
-                "network" : [">=Istanbul"],
+                "network" : ["Istanbul", "Berlin", "London"],
```

Comment:
why remove tests for nets after London?

Reply:
This test is specifically test empty account behaviour. Removing the empty account turns it into a different test, which is already implemented as `ZeroValue_TransactionCALL_OOGRevertFiller`.